go.mod

```diff
@@ -18,6 +18,8 @@ require (
 	sigs.k8s.io/controller-tools v0.11.3
 )
 
+require github.com/tidwall/gjson v1.14.4
```

Comment:
it was the `go` cli, when I move the require to the upper block in vscodium and save it, all spaces are rewrited but if it's not a problem I can try to push that

Reply:
pushed

Reply:
reverted, this lead to many git conflict

Reply:
Is there any particular reason for not using `k8s.io/client-go/util/jsonpath` instead of this library?
Also I would really like for us to have all the requires in the same grouping.